docs/UserGuide.md

```diff
@@ -129,6 +129,20 @@ java -version
 * Extraneous parameters for commands that do not take in parameters (such as `help`, `list`, `exit` and `clear`) will be ignored.<br>
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
 
+
+### Task parameters
+* A task is specified using two pieces of information: `TASK_DESCRIPTION` and `DATE TIME`.
+* `TASK_DESCRIPTION` can be any non-empty string made of alphanumeric characters.
+* `DATE TIME` must be of the form d-M-yyyy HHmm, although time can be omitted, this will result in the task being created with a default time of 0000 hours.
```

Comment:
```suggestion
* `DATE TIME` must be of the form d-M-yyyy HHmm, but the time is optional. <br>
   e.g. 2-7-2022 1345`, `28-10-2022` are valid dates.
* Although time can be omitted, this will result in the task being created with a default time of 0000 hours.
```